packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -544,3 +544,8 @@ export const signMessage = async (
   const signatures = await signer.signMessage(typedDataMessage, signerUserAddress);
   return stark.signatureToDecimalArray(signatures);
 };
+
+export const getStarkNameUtil = async (network: Network, userAddress: string) => {
+  const provider = getProvider(network);
+  return provider.getStarkName(userAddress);
```

Comment:
please check if the StarknetIdContract in sepolia
if yes, please pass in the sepolia StarknetIdContract when using sepolia

the current starknet.js does not contains  sepolia contract in getStarknetIdContract

Reply:
It's not yet deployed, but it will be in the coming days. We plan on updating starknet.js with it as well

Reply:
I made a PR to starknet.js adding the Sepolia contracts but it won't be deployed in production before a few weeks. I can add them in this PR, what do you think ?